File: Scaler/Electricity_bill.java
package Scaler;

import java.util.Scanner;

public class Electricity_bill {
    private static Scanner sc;

    public static void main(String[] args) {
        sc = new Scanner(System.in);
        System.out.print("Enter the Consume unit: ");
        int use = sc.nextInt();
        if (use < 0 && use <= 100) {
            System.out.println("No bill needs to pay");
        } else if (use > 100 && use <= 200) {
            int sum = (use - 100) * 5;
            System.out.println("Total Bill is " + sum);
        } else if (use > 200 && use <= 300) {
            int sum = (100 * 5) + ((use - 200) * 10);
            System.out.println("Total Bill is " + sum);
        } else {
            int sum = (100 * 5) + (100 * 10) + ((use - 300) * 12);
            System.out.println("Total Bill is " + sum);
        }
    }
}

File: Scaler/Electricity _bill.py
consume = int(input("Enter the consumption unit: "))
if consume>=1 and consume <=100:
    print('Total bill is zero')
if consume > 100 and consume <= 200:
    sum=(100)*5
    print(f'The total bill has to pay is {sum}')
if consume > 200 and consume <= 300:
    sum=(100)*5 + (consume-200)*10
    print(f'The total bill has to pay is {sum}')
if consume>300:
    sum=(100)*5 + (100)*10 + (consume-300)*12
    print(f'The total bill has to pay is {sum}')